planetcantile/data/generate.py

```diff
@@ -132,15 +132,14 @@ def __post_init__(self):
         pass
         #print(f'Could not find authority for {crs_obj.to_wkt()}')
 
-
-for tmsp in crss:
+for tmsp in crss[10:]:
     # create the tms object
     tms = morecantile.TileMatrixSet.custom(**asdict(tmsp))
-    tmsj = tms.json(exclude_none=True)
-    tmsj = tmsj.replace(CRS_to_uri(tms.supportedCRS), CRS_to_urn(tms.supportedCRS), 1)
-    tmsj = tmsj.replace(CRS_to_uri(tms.boundingBox.crs), CRS_to_urn(tms.boundingBox.crs), 1)
-    _d = json.loads(tmsj)  # get to pretty printed json
+    tmsj = tms.dict()
```

Comment:
I am not a fan of this change as it causes a huge diff by adding the "keywords", "title", "abstract", and "wellKnownScaleSet" json keys to output. See if it is possible to just set/override the `_geographical_crs` on the TMS object before the call to `.json`